packages/server/lib/controllers/config.controller.ts

```diff
@@ -144,11 +144,7 @@ class ConfigController {
             );
 
             res.status(200).send({
-                integrations: integrations.sort((a: Integration, b: Integration) => {
-                    const creationDateA = a.creationDate || new Date(0);
-                    const creationDateB = b.creationDate || new Date(0);
-                    return creationDateB.getTime() - creationDateA.getTime();
-                })
```

Comment:
I imagine it was to show the integrations by most recently added in the dashboard. The sorting can be done client side I think.

Reply:
yeah, actually that's what I'm changing because it's super annoying to have seemingly random order in the dashboard. 
Maybe @khaliqgant can clarify what were the expectations?

Reply:
merging, happy to revert if needed